drivers/src/hand_off.rs

```diff
@@ -275,21 +275,26 @@ pub struct FirmwareHandoffTable {
     /// Fuse log Address
     pub fuse_log_addr: u32,
 
+    /// RtAlias public key.
     pub rt_dice_pub_key: Ecc384PubKey,
 
+    /// RtAlias certificate signature.
     pub rt_dice_sign: Ecc384Signature,
 
+    /// LdevId certificate signature.
+    pub ldevid_dice_sign: Ecc384Signature,
```

Comment:
Please update the docs at https://github.com/chipsalliance/caliptra-sw/blob/main/fmc/README.md with this change.

Reply:
Done